component/addons/resourcequota.libsonnet

```
@@ -0,0 +1,29 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.prometheus;
+
+{
+  local config = self,
+  local resourcequota = {
+    apiVersion: 'v1',
+    kind: 'ResourceQuota',
+    metadata: {
+      name: 'nodeexporter-infra',
```

Might actually make sense to use the following, so that the resourcequota uses the generated name for the node-exporter instance. 

```suggestion
      name: config.values.nodeExporter.name,
```